Makefile

```diff
@@ -77,6 +77,10 @@ GOCOVXML = $(GOBIN)/gocov-xml
 $(GOBIN)/gocov-xml: | $(BASE) ; $(info  Building gocov-xml...)
 	$Q go get github.com/AlekSi/gocov-xml
 
+GCOV2LCOV = $(GOBIN)/gcov2lcov
+$(GCOV2LCOV): | $(BASE) ; $(info  building gcov2lcov...)
+	$Q go get github.com/jandelgado/gcov2lcov
```

Comment:
@zeeke will we encounter the same issues as in sriov-network-device plugin ?

(shall we change go get to go install here as well to avoid changing go.mod which might cause failure to run coverage job)

Reply:
Yes, going to fix it